### PR TITLE
Windows output dir display fix

### PR DESCRIPTION
**Pull request by tushar-tushar**
Fix directory display and service spec changes in windows system

### CODE REVIEW

File: commons/src/main/java/io/hyscale/commons/utils/OperatingSystemUtil.java [hunk 1]
```diff
@@ -16,13 +16,44 @@
...
 
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import io.hyscale.commons.config.SetupConfig;
 import io.hyscale.commons.constants.ToolConstants;
 
-public class OperatingSystemHelper {
+public class OperatingSystemUtil {
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/utils/OperatingSystemUtil.java]: This class is very specific to windows ,can we rename to some WindowsUtil ?

File: commons/src/main/java/io/hyscale/commons/utils/OperatingSystemUtil.java [hunk 2]
```diff
@@ -16,13 +16,44 @@
...
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * Update filepath to host file separator if the field is set
+	 * 
+	 */
+	public static String setWindowsFileSeparator(String filepath) {
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/utils/OperatingSystemUtil.java]: updateToHostFileSeparator() ???
it does not set the file separator directly check for windows and then sets right should be update

File: workflow-controller/controller-service/src/main/java/io/hyscale/controller/directive/impl/BuildSpecJsonHandler.java [hunk 3]
```diff
@@ -47,21 +47,21 @@ public BuildSpec updateObject(BuildSpec buildSpec) {
...
+				artifact.setSource(OperatingSystemUtil.modifyWindowsFileSeparator(artifact.getSource()));
 			});
 			buildSpec.setArtifacts(artifacts);
 		}
 
 		// Config Commands Script
 		String configCmdScript = buildSpec.getConfigCommandsScript();
 		if (StringUtils.isNotBlank(configCmdScript)) {
-			buildSpec.setConfigCommandsScript(OperatingSystemHelper.modifyWindowsFileSeparator(configCmdScript));
+			buildSpec.setConfigCommandsScript(OperatingSystemUtil.modifyWindowsFileSeparator(configCmdScript));
```
Comment (SameeraGrandhi) [inline workflow-controller/controller-service/src/main/java/io/hyscale/controller/directive/impl/BuildSpecJsonHandler.java]: Can we rename this method modifyWindowsFileSeparator ?

Comment (tushar-tushar) [reply workflow-controller/controller-service/src/main/java/io/hyscale/controller/directive/impl/BuildSpecJsonHandler.java]: Something like updateToUnixFileSeparator?

**Pull request merged**